packages/legend-query-builder/src/stores/filter/QueryBuilderFilterState.ts

```diff
@@ -596,17 +630,6 @@ export class QueryBuilderFilterTreeConditionNodeData
     this.isNewlyAdded = val;
   }
 
-  get isExistsNodeChild(): boolean {
```

Comment:
Moved this to be a function instead of a class method to be more memory efficient, so that every instance of the class doesn't have a copy of this function.